packages/protocol/contracts/L1/verifiers/SgxVerifier.sol

```diff
@@ -19,6 +19,8 @@ import "../../common/EssentialContract.sol";
 import "../../thirdparty/LibBytesUtils.sol";
 import "../ITaikoL1.sol";
 import "./IVerifier.sol";
+import { IAttestation } from "../../thirdparty/onchainRA/interfaces/IAttestation.sol";
```

Comment:
as you can see in other files, we basically switched away from using `import {...} from xxx` to `import xxx` unless you want to rename the imported contracts